cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/RenameLogic.java

```diff
@@ -36,14 +54,13 @@ public void renameApplication(ApplicationsOperations applicationsOperations, Str
 
     /**
      * Rename an existing service.
-     * @param servicesOperations ServicesOperations
      * @param newName The new name of the service
      * @param currentName The current name of the service
      * @throws UpdateException if an error occurs during the nameChange procedure
      * @throws NullPointerException when newName, currentName or servicesOperations is null
```

Comment:
The servicesOperations are no longer passed:
```suggestion
     * @throws NullPointerException if any of the arguments is null
```